Dev7/EquilateralTriangle.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Extends from Triangle, creates an equilateral triangle if it exists.
```

Comment:
Wrong comment as the class represents equilateral triangle not creates it (you create class instance by operator new)